specs/cm/change-mgt-shapes.html

```diff
@@ -31,7 +31,7 @@
       var respecConfig = {
         specStatus: specStatus,
         revision: revision,
-        //publishDate: "2019-08-15",
+        publishDate: "2020-09-17",
         previousPublishDate: "2018-08-24",
         previousMaturity: "CS",
```

Comment:
is this true? I guess there was a PSD first.

Reply:
It was a CS, migrating to a PS

Reply:
huh? I edited PSD03 https://docs.oasis-open-projects.org/oslc-op/cm/v3.0/psd03/change-mgt-spec.html

Reply:
Don't think it ever got published because of other changes we decided to do. I jus followed the OASIS latest link to see what was there.